packages/pyolite-kernel/src/kernel.ts

```diff
@@ -10,6 +10,13 @@ import worker from './worker?raw';
 
 import { PIPLITE_WHEEL } from './_pypi';
 
+declare global {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  interface Window {
+    crossOriginIsolated: boolean | undefined;
```

Comment:
can we find a more canonical source for this type? or at least document where this magic variable comes from. At the very least, we'd want to move it out of the implementation, and probably in a dedicated `.d.ts` somewhere...

Reply:
I'll move this to the declarations.d.ts for now. Types for crossOriginIsolated were added in typescript 4.4. https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1029

I'm happy to find another way to do this as well, if you have any suggestions.

Reply:
Looks like [upstream](https://github.com/jupyterlab/jupyterlab/pull/11594) is on:

```
 "typescript": "~4.5.2",
```

This would probably be separate PR, and we should have a look at what other goodies we can get any new from static analysis tooling.